src/pages/Main/LoginAside/LoginAside.js

```diff
@@ -0,0 +1,43 @@
+import { Link } from 'react-router-dom';
+import './LoginAside.scss';
+
+export default function LoginAside() {
+  return (
+    <div className="loginAside">
+      <img alt="닫기버튼" src="" className="closeBtn" />
+      <div className="account">
+        <h1>Hej</h1>
+        <span />
+        <Link to="/login">
+          <button type="button" className="loginBtn">
+            로그인
+          </button>
+        </Link>
+      </div>
+      <div className="sign">
+        <Link to="/signup" className="signLink">
+          <div className="signComponent ">
+            {/*컴포넌트 재사용*/}
```

Comment:
머지한 이후 추후 추가 기능 구현 때 수정할 내용이면,
```suggestion
            {/*TODO: 컴포넌트 재사용 적용해서 수정하기*/}
```
이런식으로 컨벤션을 정해서 주석 달아두기
ex) `// TODO:`, `// FIXME:`